doc/reference/shelley-genesis.md

```diff
@@ -1295,26 +1394,47 @@ We will be using the low level `build-raw` command because a more convenient
-                                                 --ttl SLOT_COUNT --fee LOVELACE
-                                                 --tx-body-file FILEPATH
-                                                 [--certificate FILEPATH]
+Usage: cardano-cli shelley transaction build-raw --tx-in TX-IN --tx-out TX-OUT
+                                                 --ttl SLOT --fee LOVELACE
+                                                 [--certificate-file FILE]
+                                                 [--withdrawal WITHDRAWAL]
+                                                 [--json-metadata-no-schema |
+                                                   --json-metadata-detailed-schema]
+                                                 [--metadata-json-file FILE |
+                                                   --metadata-cbor-file FILE]
+                                                 [--update-proposal-file FILE]
+                                                 --out-file FILE
   Build a transaction (low-level, inconvenient)
 
 Available options:
-  --tx-in TX_IN            The input transaction as TxId#TxIx where TxId is the
+  --tx-in TX-IN            The input transaction as TxId#TxIx where TxId is the
                            transaction hash and TxIx is the index.
-  --tx-out TX_OUT          The ouput transaction as TxOut+Lovelace where TxOut
-                           is the hex encoded address followed by the amount in
-                           Lovelace.
-  --ttl SLOT_COUNT         Time to live (in slots).
+  --tx-out TX-OUT          The ouput transaction as Address+Lovelace where
+                           Address is the Bech32-encoded address followed by the
+                           amount in Lovelace.
+  --ttl SLOT               Time to live (in slots).
   --fee LOVELACE           The fee amount in Lovelace.
-  --tx-body-file FILEPATH  Output filepath of the TxBody.
-  --certificate FILEPATH   Filepath of the certificate. This encompasses all
+  --certificate-file FILE  Filepath of the certificate. This encompasses all
                            types of certificates (stake pool certificates, stake
                            key certificates etc)
+  --withdrawal WITHDRAWAL  The reward withdrawal as StakeAddress+Lovelace where
+                           StakeAddress is the Bech32-encoded stake address
+                           followed by the amount in Lovelace.
+  --json-metadata-no-schema
+                           Use the "no schema" conversion from JSON to tx
+                           metadata.
+  --json-metadata-detailed-schema
+                           Use the "detailed schema" conversion from JSON to tx
+                           metadata.
+  --metadata-json-file FILE
+                           Filepath of the metadata file, in JSON format.
+  --metadata-cbor-file FILE
+                           Filepath of the metadata, in raw CBOR format.
```

Comment:
Should this also say "metadata file"?

Reply:
That description for `metadata-json-file` should probably be changed, actually. "Filepath of the metadata file" sounds redundant.